File: TransCoder_tokenized_test_set_functions/python/PRINT_DISTINCT_ELEMENTS_GIVEN_INTEGER_ARRAY_1.py
def printDistinct ( arr, n ) :
    arr . sort ( );
    
    for i in range ( n ) :
        if ( i < n - 1 and arr [ i ] == arr [ i + 1 ] ) :
            while ( i < n - 1 and ( arr [ i ] == arr [ i + 1 ] ) ) :
                i += 1;
                
        else :
            print ( arr [ i ], end = " " );
            
    

File: TransCoder_tokenized_test_set_functions/java/PRINT_DISTINCT_ELEMENTS_GIVEN_INTEGER_ARRAY_1.java
static void printDistinct ( int arr [ ], int n ) {
  Arrays . sort ( arr );
  for ( int i = 0;
  i < n;
  i ++ ) {
    while ( i < n - 1 && arr [ i ] == arr [ i + 1 ] ) i ++;
    System . out . print ( arr [ i ] + " " );
  }
}